packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,111 @@
+# MDC Toolbar
+
+MDC Toolbar provides an `mdc-toolbar` class for easily styling a RTL-aware
```

Comment:
We usually try and link to the spec that each component corresponds to within the top of the README, e.g. 

> MDC Toolbar provides a Material Design toolbar component adhering to the [Material Design toolbar spec](https://material.io/guidelines/components/toolbars.html#)

Furthermore, we should probably make a note that this does not yet support the "Waterfall" or "Flexible Header" patterns